CMakeLists.txt

```diff
@@ -66,6 +66,17 @@ if(NOT CMAKE_BUILD_TYPE IN_LIST BUILD_TYPES)
     message(FATAL_ERROR "Unknown Build Type: ${CMAKE_BUILD_TYPE}")
 endif()
 
+# Flag to enforce colored output. When compiler recognize output to terminals, recent version automatically
+# set colored output. However, when using Ninja, the output is not directly the terminal. With this flag enabled,
```

Comment:
< the output is not directly the terminal

This is easily misunderstood. Maybe something like "directly sent to the terminal"?